cli/submit.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import json
+import requests
+from getpass import getpass
+
+
+class StudySubmitter:
+    def __init__(self, ENA_AUTH_URL="https://www.ebi.ac.uk/ena/submit/webin/auth/token",
+                 WEBIN_SUBMIT_ENDPOINT ="http://www.ebi.ac.uk/eva/v1/submission/initiate/webin"):
+        self.ENA_AUTH_URL = ENA_AUTH_URL
+        self.WEBIN_SUBMIT_ENDPOINT = WEBIN_SUBMIT_ENDPOINT
+
+    # TODO
+    def submit_with_lsri_auth(self):
+        return NotImplementedError
```

Comment:
I would naively assume that submission with LSRI would be a different class. 
I guess that can be refactored later.